OpenRA.Mods.Common/UtilityCommands/ExtractLuaDocsCommand.cs

```diff
@@ -77,6 +77,9 @@ void IUtilityCommand.Run(Utility utility, string[] args)
 				foreach (var m in members.OrderBy(m => m.Name))
 				{
 					var desc = Utility.HasAttribute<DescAttribute>(m) ? Utility.GetCustomAttributes<DescAttribute>(m, true).First().Lines.JoinWith("<br />") : "";
+					if (Utility.HasAttribute<ObsoleteAttribute>(m))
+						desc += $"<br />**[DEPRECATED] {Utility.GetCustomAttributes<ObsoleteAttribute>(m, true).First().Message}**";
```

Comment:
```suggestion
						desc += $"<br />**Deprecated: {Utility.GetCustomAttributes<ObsoleteAttribute>(m, true).First().Message}**";
```

Reply:
Ok, but why make it less visible? My goal was to make it as obvious as possible 🤷🏻‍♂️

Reply:
The strike through makes it even more visible. The CAPS LOCK and the square brackets were a bit odd.